src/utils/restUtils.ts

```diff
@@ -1,4 +1,7 @@
 export function erProd(): boolean {
+    if (typeof window === "undefined") {
```

Comment:
Siden nextjs kjører på serversiden har vi ikke tilgang til `window` objektet. Legger derfor inn en ekstra sjekk her og defaulter til `true` slik at vi ikke skal påvirke koden for dagens veiviser. På sikt er planen å legge til ENV-variabler for de tilfellene hvor vi vil skille på test og prod.